sources/vega/vega.proto

```diff
@@ -938,6 +938,24 @@ message NetworkParameter {
   string value = 2;
 }
 
+// Network limits, defined in the genesis file
+message NetworkLimits {
+  // Are market proposals allowed at this point in time
+	bool can_propose_market = 1;
+  // Are asset proposals allowed at this point in time
+  bool can_propose_asset = 2;
+  // True once block count > bootstrapBlockCount
+  bool bootstrap_finished = 3;
+  // Are market proposals enabled on this chain
+	bool propose_market_enabled = 4;
+  // Are asset proposals enabled on this chain
+  bool propose_asset_enabled = 5;
+  // How many blocks before the chain comes out of bootstrap mode
+	uint32 bootstrap_block_count = 6;
+  // True once the genesis file is loaded
+	bool genesis_loaded = 7;
+}
```

Comment:
formatting looks weird-ish here.

Reply:
Sorry will fix - it's tabs vs spaces, they line up nicely in my editor :) VS code is usually quite good at figuring out what's already there and copying it but obviously not here!